dsgrid/dataset/dataset_schema_handler_base.py

```diff
@@ -103,8 +103,9 @@ def get_pivot_dimension_columns_mapped_to_project(self):
         """
         columns = set(self.get_pivot_dimension_columns())
         dim_type = self.get_pivot_dimension_type()
+        mapped_pivot_columns = columns
         for ref in self._mapping_references:
-            if ref.from_dimension_type.value == dim_type:
+            if ref.from_dimension_type == dim_type:
```

Comment:
This doesn't check for multiple matches, which, if it occurred, it would be a bug. We need to identify a place to check for that. Assuming there is only one match, this if clause could end with a `break`.